src/Scylla/interceptors.ts

```diff
@@ -0,0 +1,23 @@
+import { PaperbackInterceptor, Request, Response } from "@paperback/types";
+import { SCYLLA_DOMAIN } from "./main";
+
+export class ScyllaInterceptor extends PaperbackInterceptor {
+    async interceptRequest(request: Request): Promise<Request> {
+        request.headers = {
+            ...(request.headers ?? {}),
+            ...{
+                referer: `${SCYLLA_DOMAIN}/`,
+                "user-agent": await Application.getDefaultUserAgent(),
+            },
+        };
+        return request;
+    }
```

Comment:
Setting the user agent to the default should not be necessary
```suggestion
    async interceptRequest(request: Request): Promise<Request> {
        request.headers.referer = `${SCYLLA_DOMAIN}/`;

        return request;
    }
```